### PR TITLE
input_devices_plug_unplug: add case

### PR DESCRIPTION
**Pull request by dzhengfy**
Test input device, like tablet, mouse, keyboard,  hotplug and hotunplug with virtio and usb bus.
Case ID: xxx-117594

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
libvirt-9.5.0
```
 (1/1) type_specific.io-github-autotest-libvirt.virtual_devices.input_devices_plug_unplug.hot.bus_type_virtio.tablet_mouse_keyboard: WARN: Test passed but there were warnings during execution. Check the log for details. (46.82 s)
 (1/1) type_specific.io-github-autotest-libvirt.virtual_devices.input_devices_plug_unplug.hot.bus_type_usb.tablet_mouse_keyboard: WARN: Test passed but there were warnings during execution. Check the log for details. (47.01 s)
```

**Comment by dzhengfy** [comment]
@zhentang-tz , Could you help review?

File: libvirt/tests/cfg/virtual_device/input_devices_plug_unplug.cfg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- virtual_devices.input_devices_plug_unplug:
+    type = input_devices_plug_unplug
+    take_regular_screendumps = no
+    start_vm = no
```
Comment (zhentang-tz) [inline libvirt/tests/cfg/virtual_device/input_devices_plug_unplug.cfg:4]: What does this config used for?

Comment (dzhengfy) [reply libvirt/tests/cfg/virtual_device/input_devices_plug_unplug.cfg:4]: It means the vm will not be started by the avocado framework before our test script is executed.

**Review by zhentang-tz — APPROVED** [review]
looks good to me

**Review by Yingshun — APPROVED** [review]

**Pull request merged**